### ISSUE TITLE
Unable to save files

### ISSUE THREAD

**Issue by randycoulman** [rgreenjr/atom-elixir-formatter#15]
Atom version: 1.23.3
atom-elixir-formatter version: 1.0.0

To reproduce:
- Set Format on Save option to `Only if project includes a .formatter.exs file`
- Open a project that has a `.formatter.exs` file
- Open a file from outside the project.  Choosing `Atom | Config...` from Atom's menu is a good option
- Make a simple change and attempt to save
- Notice that the file is not saved

My limited testing shows that saving non-Elixir files within the project tree works fine.

The console in Atom shows the following error:

```
Uncaught (in promise) TypeError: Path must be a string. Received null
    at assertPath (path.js:7:11)
    at join (path.js:1211:7)
    at Object.isFormatterFilePresent (/Users/randy/.dotfiles/atom/packages/atom-elixir-formatter/lib/settings.js:33:7)
    at Object.shouldFormatOnSave (/Users/randy/.dotfiles/atom/packages/atom-elixir-formatter/lib/settings.js:44:19)
    at /Users/randy/.dotfiles/atom/packages/atom-elixir-formatter/lib/main.js:42:18
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.<anonymous> (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:154:50)
    at Emitter.module.exports.Emitter.emitAsync (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:157:18)
    at /Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:1283:38
```

**Comment by rgreenjr** [comment]
I'm out of town until Sunday, but will look into this as soon as I get back. Thanks for detailed bug report.

**Comment by rgreenjr** [comment]
Hi @randycoulman – I just published version 1.0.1 with a fix for this bug. Please take a look and let me know if it works for you. Thanks.

**Comment by randycoulman** [comment]
Yup, sure seems to.  Thanks!
